app/controllers/submit.js

```diff
@@ -333,8 +338,38 @@ export default Ember.Controller.extend(BasicsValidations, NodeActionsMixin, Tagg
             this.send('next', this.get('_names.0'));
         },
         resetFileUpload() {
-            this.send('changeState', State.START);
-            this.set('fileAndNodeLocked', false);
+            var promisesArray = [];
+            var filePromises = [];
+            this.get('filesUploadedForPreprint').forEach((file) => {
+                filePromises.push(this.get('fileManager').deleteFile(file).then(() => {
+                    this.get('toast').info('File removed!');
+                }).catch(() => {
+                    this.get('toast').error('Could not remove uploaded file');
+                }));
+            });
+
+            Ember.RSVP.all(filePromises).then(() => {
```

Comment:
I hadn't looked into RSVP.allSettled - thanks - I'll look more into this in the AM.